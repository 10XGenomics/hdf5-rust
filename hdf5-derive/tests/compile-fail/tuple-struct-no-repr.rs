use hdf5_metno_derive::H5Type;

#[derive(H5Type)]
//~^ ERROR proc-macro derive
//~^^ HELP H5Type requires repr(C), repr(packed) or repr(transparent) for tuple structs
struct Foo(i64);

fn main() {}
