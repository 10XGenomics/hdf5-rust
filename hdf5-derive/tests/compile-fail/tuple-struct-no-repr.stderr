error: `H5Type` requires repr(C), repr(packed) or repr(transparent) for tuple structs
 --> $DIR/tuple-struct-no-repr.rs:7:8
  |
7 | struct Foo(i64);
  |        ^^^
